### DISCUSSION TITLE
MQTT and I robot create 3

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#421] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

jetson agx orin

### Which version of ROS 2 is installed on your computer?

Galactic

### Which firmware version is installed on your robot?

G 4.5.5

### Which RMW is your robot running?

CycloneDDS

### Is the robot connected to a network? If so, what is the network type?

home

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

Bluetooth (default)

### Describe your question.

Is it possible to have I robot create 3 send and receive    telemetry data from a MQTT server
What i am trying to do is have all the telemetry data go to a MQTT server and then have  other devices  query the MQTT server 
To grab data that it needs.
What got me thinking about it was I was going through the logs of the I robot create 3 and found it was connecting to a mosquito MQTT server. 

here is the info from the LOG file on the I robot create 3:

`iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886602: New connection from 127.0.0.1 on port 1883.
Jul  9 07:10:02 iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886602: New client connected from 127.0.0.1 as mosqpub/18669-iRobot-87 (c1, k60).
Jul  9 07:10:02 iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886602: Client mosqpub/18669-iRobot-87 disconnected.
Jul  9 07:10:28 iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886628: New connection from 127.0.0.1 on port 1883.
Jul  9 07:10:28 iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886628: New client connected from 127.0.0.1 as mosqpub/18737-iRobot-87 (c1, k60).
Jul  9 07:10:28 iRobot-87CDD4E885C34172B233E051D0930254 user.notice mosquitto: 1688886628: Client mosqpub/18737-iRobot-87 disconnected.`

**Comment by justinIRBT** [comment]
This isn't on our roadmap.  Perhaps you could run a ROS2 telemetry to MQTT conversion on your Jetson and run the MQTT server on your Jetson.
